woodchipper/configs.py

```diff
@@ -60,7 +60,7 @@ class DevLogToStdout(BaseConfigClass):
         structlog.processors.StackInfoRenderer(),
         structlog.dev.set_exc_info,
         structlog.processors.UnicodeDecoder(),
-        structlog.processors.TimeStamper(fmt="%Y-%m-%d %H:%M.%S", utc=False),
+        structlog.processors.TimeStamper(fmt="iso", utc=False),
```

Comment:
What value will this actually output? ISO 8601 allows `Z` endings for UTC which has interop problems with python stdlib parsing functions.